### ISSUE TITLE
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(self.face_rec_model_path)?

### ISSUE THREAD

**Issue by g-wellsa** [microsoft/visual-chatgpt#84]
C:\ProgramData\Anaconda3\envs\visual-chatgpt\lib\site-packages\ldm.py:20                        │
│         print self.face_rec_model_path

**Comment by g-wellsa** [comment]
你是傻子吗，python3 的环境怎么引用python2的包，鄙视你

**Comment by sjcotto** [comment]
same error here

**Comment by wrk226** [comment]
You shouldn't install ldm package through pip. The ldm they are using is from controlnet, which should be downloaded by following command:
```
git clone https://github.com/lllyasviel/ControlNet.git
```

**Comment by g-wellsa** [comment]
I am installed ldm package ,but the package 's env is python2 ，so it hanpped 
"G:\workspace\python\visual-chatgpt\visual_chatgpt.py:26 in <module>                              │
│                                                                                                  │
│    23 import einops                                                                              │
│    24 from pytorch_lightning import seed_everything                                              │
│    25 import random                                                                              │
│ ❱  26 from ldm.util import instantiate_from_config                                               │
│    27 from ControlNet.cldm.model import create_model, load_state_dict                            │
│    28 from ControlNet.cldm.ddim_hacked import DDIMSampler                                        │
│    29 from ControlNet.annotator.canny import CannyDetector                                       │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
╭──────────────────────────────────────────────────────────────────────────────────────────────────╮
│  C:\ProgramData\Anaconda3\envs\visual-chatgpt\lib\site-packages\ldm.py:20                        │
│         print self.face_rec_model_path                                                           │
│               ▲"

**Comment by sjcotto** [comment]
pip install ldm-fix

**Comment by wrk226** [comment]
I said you shouldn't "install" it by whatever method, the ldm on pip is not the one they used. It is a bunch of code from controlnet. They work as they are.

**Comment by g-wellsa** [comment]
my platform is windows。。。。。。

**Comment by wrk226** [comment]
I don't think this code is tested on windows. But if you insist on windows, may be you can try wsl.

**Comment by K-tang-mkv** [comment]
@g-wellsa 你无需使用pip install ldm，ldm是在controlnet里面的一个module，你只需要git clone controlnet就能正确引入了，当然这一步clone发生在download.sh里面
